changelog/6771.feature.rst

```diff
@@ -0,0 +1 @@
+Added an example to find the location of STEREO-A using `sunpy` and `solarmach`
```

Comment:
For this, you will want to link (this refers to sphinx links and not URLS) the two examples here. 

You will want to check other example pull requests to see how its done.